le_utils/constants/content_kinds.py

```diff
@@ -32,6 +34,7 @@
     file_formats.EPUB: DOCUMENT,
     file_formats.PERSEUS: EXERCISE,
     file_formats.HTML5: HTML5,
+    file_formats.SLIDESHOW: SLIDESHOW,
```

Comment:
TODO: discuss

Reply:
I feel like `IMAGE` might be better here, and doesn't tie us too tightly to our implementation choices.